cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -5,46 +5,488 @@
             }
+
+            &-link {
+                padding-bottom: @grid_gutter-width;
+                margin-right: @grid_gutter-width / 2;
+                margin-left: @grid_gutter-width / 2;
+
+                font-size: unit( 18px / @base-font-size-px, em );
+
+                &:first-child {
+                    margin-left: 0;
+                }
+
+                // TODO: Show menu on hover and remove :hover selector.
+                &[aria-expanded="true"],
+                &__current,
+                &:hover {
+                    padding-bottom: @grid_gutter-width - 6px;
+                    border-bottom: 6px solid @black;
+                }
+            }
+        }
+
+        &_subcontent {
+            overflow: hidden;
+            position: absolute;
+            left: 0;
+            z-index: 10;
+            width: 100%;
+
+            // Set an arbtrary min-height to create a "window" into which
+            // the menu will slide into view.
+            min-height: 600px;
+            pointer-events: none;
+
+            &-wrapper {
+                pointer-events: auto;
+                background-color: @gray-5;
+                border-top: 1px solid @gray-50;
+                border-bottom: 1px solid @gray-50;
+                transform: translateY( -100% );
+                transition: transform 0.25s ease-out;
+
+                .u-drop-shadow-after();
+
+                .wrapper {
+                    padding: @grid_gutter-width;
+                }
+            }
+
+            &[aria-expanded="true"] {
+
+                .o-mega-menu_subcontent-wrapper {
+                    display: block;
+                    transform: translateY( 0 );
+                }
+            }
+
+            &-btn__back {
+                display: none;
+            }
+
+            &-title {
+                padding-bottom: @grid_gutter-width / 2;
+                margin-bottom: unit( @grid_gutter-width / 2 / 18px, em );
+
+                font-size: 1em;
+            }
+
+            &-list {
+                .grid_column(4);
+
+                &:first-child {
+                    border-left: 0;
+                }
+
+                &:last-child {
+                    border-right: 0;
+                }
+            }
+
+            .list_link {
+                padding-top: unit( 10px / 18px, em );
+                padding-bottom: unit( 10px / 18px, em );
+                padding-left: unit( 15px / 18px, em );
+
+                border-left: 5px solid transparent;
+                font-size: unit( 18px / @base-font-size-px, em );
+
+                &:hover,
+                &__current {
+                    border-left-color: @green;
+                }
+
+                &__current {
+                    cursor: default;
+                }
+            }
+        }
+
+        // Hide global header CTA and eyebrow at desktop size.
+        .m-global-header-cta,
+        .m-global-eyebrow {
+            display: none;
```

Comment:
Is there value in having all the desktop stuff wrapped in `.respond-to-min( @bp-med-min, {`? I kind of like the organization of not moving desktop-specific stuff outside of the breakpoint. Also, it seems like the mega-menu should just care that it's not displaying the global header cta and eyebrow at desktop sizes. It doesn't care if they're in a list or horizontal layout. E.g. if the horizontal layout ended up being used in the mega-menu for some reason it shouldn't require an update here to re-hide it at desktop sizes.